core/src/main/java/org/infinispan/statetransfer/StateTransferManagerImpl.java

```diff
@@ -122,6 +125,20 @@ public void rebalance(CacheTopology cacheTopology) {
       }
    }
 
+   private Optional<Float> globalCapacityFactor() {
+      String globalCapacityFactorProperty = System.getProperty(GLOBAL_CAPACITY_FACTOR);
```

Comment:
As Tristan said, this should be an on-off switch, and it would be better to make it a global setting protected by a feature flag.